2024-10-19 23:33:16,413 INFO    MainThread:3279171 [wandb_setup.py:_flush():79] Current SDK version is 0.18.5
2024-10-19 23:33:16,413 INFO    MainThread:3279171 [wandb_setup.py:_flush():79] Configure stats pid to 3279171
2024-10-19 23:33:16,413 INFO    MainThread:3279171 [wandb_setup.py:_flush():79] Loading settings from /home/gcpuser/.config/wandb/settings
2024-10-19 23:33:16,413 INFO    MainThread:3279171 [wandb_setup.py:_flush():79] Loading settings from /home/gcpuser/gene/wandb/settings
2024-10-19 23:33:16,413 INFO    MainThread:3279171 [wandb_setup.py:_flush():79] Loading settings from environment variables: {'api_key': '***REDACTED***'}
2024-10-19 23:33:16,414 INFO    MainThread:3279171 [wandb_setup.py:_flush():79] Applying setup settings: {'mode': None, '_disable_service': None}
2024-10-19 23:33:16,414 INFO    MainThread:3279171 [wandb_setup.py:_flush():79] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-10-19 23:33:16,414 INFO    MainThread:3279171 [wandb_setup.py:_flush():79] Applying login settings: {}
2024-10-19 23:33:16,414 INFO    MainThread:3279171 [wandb_init.py:_log_setup():534] Logging user logs to /home/gcpuser/gene/wandb/run-20241019_233316-sj91o0gr/logs/debug.log
2024-10-19 23:33:16,414 INFO    MainThread:3279171 [wandb_init.py:_log_setup():535] Logging internal logs to /home/gcpuser/gene/wandb/run-20241019_233316-sj91o0gr/logs/debug-internal.log
2024-10-19 23:33:16,414 INFO    MainThread:3279171 [wandb_init.py:init():621] calling init triggers
2024-10-19 23:33:16,414 INFO    MainThread:3279171 [wandb_init.py:init():628] wandb.init called with sweep_config: {}
config: {}
2024-10-19 23:33:16,414 INFO    MainThread:3279171 [wandb_init.py:init():646] re-initializing run, found existing run on stack: pms50wod
2024-10-19 23:33:16,416 INFO    MainThread:3279171 [wandb_run.py:_finish():2158] finishing run michaelvll/gene/pms50wod
2024-10-19 23:33:16,417 ERROR   MainThread:3279171 [jupyter.py:save_history():451] Run pip install nbformat to save notebook history
2024-10-19 23:33:16,417 INFO    MainThread:3279171 [jupyter.py:save_ipynb():387] not saving jupyter notebook
2024-10-19 23:33:16,417 INFO    MainThread:3279171 [wandb_init.py:_jupyter_teardown():462] cleaning up jupyter logic
2024-10-19 23:33:16,417 INFO    MainThread:3279171 [wandb_run.py:_atexit_cleanup():2426] got exitcode: 0
2024-10-19 23:33:16,417 INFO    MainThread:3279171 [wandb_run.py:_restore():2408] restore
2024-10-19 23:33:16,418 INFO    MainThread:3279171 [wandb_run.py:_restore():2414] restore done
2024-10-19 23:33:17,844 INFO    MainThread:3279171 [wandb_run.py:_footer_history_summary_info():3975] rendering history
2024-10-19 23:33:17,845 INFO    MainThread:3279171 [wandb_run.py:_footer_history_summary_info():4007] rendering summary
2024-10-19 23:33:17,852 INFO    MainThread:3279171 [wandb_run.py:_footer_sync_info():3934] logging synced files
2024-10-19 23:33:17,857 INFO    MainThread:3279171 [wandb_init.py:init():671] starting backend
2024-10-19 23:33:17,857 INFO    MainThread:3279171 [wandb_init.py:init():675] sending inform_init request
2024-10-19 23:33:17,859 INFO    MainThread:3279171 [backend.py:_multiprocessing_setup():104] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-10-19 23:33:17,859 INFO    MainThread:3279171 [wandb_init.py:init():688] backend started and connected
2024-10-19 23:33:17,862 INFO    MainThread:3279171 [wandb_run.py:_label_probe_notebook():1342] probe notebook
2024-10-19 23:33:17,862 INFO    MainThread:3279171 [wandb_init.py:init():783] updated telemetry
2024-10-19 23:33:17,866 INFO    MainThread:3279171 [wandb_init.py:init():816] communicating run to backend with 90.0 second timeout
2024-10-19 23:33:18,900 INFO    MainThread:3279171 [wandb_init.py:init():867] starting run threads in backend
2024-10-19 23:33:18,994 INFO    MainThread:3279171 [wandb_run.py:_console_start():2463] atexit reg
2024-10-19 23:33:18,994 INFO    MainThread:3279171 [wandb_run.py:_redirect():2311] redirect: wrap_raw
2024-10-19 23:33:18,995 INFO    MainThread:3279171 [wandb_run.py:_redirect():2376] Wrapping output streams.
2024-10-19 23:33:18,995 INFO    MainThread:3279171 [wandb_run.py:_redirect():2401] Redirects installed.
2024-10-19 23:33:18,995 INFO    MainThread:3279171 [wandb_init.py:init():911] run started, returning control to user process
2024-10-19 23:33:27,975 INFO    MainThread:3279171 [wandb_setup.py:_flush():79] Current SDK version is 0.18.5
2024-10-19 23:33:27,976 INFO    MainThread:3279171 [wandb_setup.py:_flush():79] Configure stats pid to 3279171
2024-10-19 23:33:27,976 INFO    MainThread:3279171 [wandb_setup.py:_flush():79] Loading settings from /home/gcpuser/.config/wandb/settings
2024-10-19 23:33:27,976 INFO    MainThread:3279171 [wandb_setup.py:_flush():79] Loading settings from /home/gcpuser/gene/wandb/settings
2024-10-19 23:33:27,976 INFO    MainThread:3279171 [wandb_setup.py:_flush():79] Loading settings from environment variables: {'api_key': '***REDACTED***'}
2024-10-19 23:33:27,976 INFO    MainThread:3279171 [wandb_setup.py:_flush():79] Applying setup settings: {'mode': None, '_disable_service': None}
2024-10-19 23:33:27,976 INFO    MainThread:3279171 [wandb_setup.py:_flush():79] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-10-19 23:33:27,976 INFO    MainThread:3279171 [wandb_setup.py:_flush():79] Applying login settings: {}
2024-10-19 23:33:27,976 INFO    MainThread:3279171 [wandb_init.py:_log_setup():534] Logging user logs to /home/gcpuser/gene/wandb/run-20241019_233327-o9i997o9/logs/debug.log
2024-10-19 23:33:27,977 INFO    MainThread:3279171 [wandb_init.py:_log_setup():535] Logging internal logs to /home/gcpuser/gene/wandb/run-20241019_233327-o9i997o9/logs/debug-internal.log
2024-10-19 23:33:27,977 INFO    MainThread:3279171 [wandb_init.py:_jupyter_setup():480] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f89af35eaa0>
2024-10-19 23:33:27,977 INFO    MainThread:3279171 [wandb_init.py:init():621] calling init triggers
2024-10-19 23:33:27,977 INFO    MainThread:3279171 [wandb_init.py:init():628] wandb.init called with sweep_config: {}
config: {}
2024-10-19 23:33:27,977 INFO    MainThread:3279171 [wandb_init.py:init():646] re-initializing run, found existing run on stack: sj91o0gr
2024-10-19 23:33:27,979 INFO    MainThread:3279171 [wandb_run.py:_finish():2158] finishing run michaelvll/gene/sj91o0gr
2024-10-19 23:33:27,980 INFO    MainThread:3279171 [wandb_run.py:_atexit_cleanup():2426] got exitcode: 0
2024-10-19 23:33:27,980 INFO    MainThread:3279171 [wandb_run.py:_restore():2408] restore
2024-10-19 23:33:27,980 INFO    MainThread:3279171 [wandb_run.py:_restore():2414] restore done
2024-10-19 23:33:29,252 INFO    MainThread:3279171 [wandb_run.py:_footer_history_summary_info():3975] rendering history
2024-10-19 23:33:29,253 INFO    MainThread:3279171 [wandb_run.py:_footer_history_summary_info():4007] rendering summary
2024-10-19 23:33:29,259 INFO    MainThread:3279171 [wandb_run.py:_footer_sync_info():3934] logging synced files
